modules/navier_stokes/doc/content/source/postprocessors/IntegralDirectedSurfaceForce.md

```diff
@@ -0,0 +1,26 @@
+# IntegralDirectedSurfaceForce
+
+!syntax description /Postprocessors/IntegralDirectedSurfaceForce
+
+## Explanation
+
+This postprocessor which computes the directed force coming from friction and pressure
+differences on a surface defined as:
+
+!equation
+F_d = \int\limits_S (\sigma \vec{n}) \cdot \vec{d} ~dS
+
+where $\sigma$ is the Cauchy stress tensor. One can use this in combination with other postprocessors
+to compute representative drag and lift coefficients.
+
+## Example input syntax
+
+In this case, the lift and drag coefficients are computed on the surface of a cylinder.
```

Comment:
Can you explain a little about the sign and vector choices of the `principal_direction` in the example? E.g. why do we choose the -x direction for calculation of the drag force? And -y direction for lift? This is probably obvious to some people, but it's not to me

Reply:
Ahhh I almost forgot this, I wanted to follow the description in this:
https://www.ljll.math.upmc.fr/hecht/ftp/NSF03/old/2014-15/w8/SchaeferTurek1996.pdf

But I added a negative sign wrongly due to a false perception of the cauchy stress tensor acting on the fluid. But we are actually acting on the solid (counter-force).